sql/snowflake/SnowflakeLexer.g4

```diff
@@ -413,6 +413,7 @@ IS:                                                    'IS';
 ISNULL:                                                'ISNULL';
 ISNUMERIC:                                             'ISNUMERIC';
 ISOLATION:                                             'ISOLATION';
+JAVASCRIPT:                                            'JAVASCRIPT';
```

Comment:
I guess it should be added to [non_reserved_words](https://github.com/antlr/grammars-v4/blob/master/sql/snowflake/SnowflakeParser.g4#L3267) since it can be identifier.